axelor-account/src/main/java/com/axelor/apps/account/service/analytic/AnalyticMoveLineServiceImpl.java

```diff
@@ -327,15 +328,18 @@ public AnalyticMoveLine generateAnalyticMoveLine(
   }
 
   @Override
-  public String getAnalyticAxisDomain(AnalyticMoveLine analyticMoveLine, Company company) {
+  public String getAnalyticAxisDomain(Company company) {
     StringBuilder domain = new StringBuilder();
     domain.append("(self.company is null OR self.company.id = " + company.getId() + ")");
-    List<AnalyticAxis> analyticAxisList =
-        accountConfigRepository.findByCompany(company).getAnalyticAxisByCompanyList().stream()
-            .map(AnalyticAxisByCompany::getAnalyticAxis)
-            .collect(Collectors.toList());
-    String idList = StringTool.getIdListString(analyticAxisList);
-    domain.append(" AND self.id IN (" + idList + ")");
+    AccountConfig accountConfig = accountConfigRepository.findByCompany(company);
+    if (accountConfig != null) {
+      List<AnalyticAxis> analyticAxisList =
+          accountConfig.getAnalyticAxisByCompanyList().stream()
+              .map(AnalyticAxisByCompany::getAnalyticAxis)
+              .collect(Collectors.toList());
+      String idList = StringTool.getIdListString(analyticAxisList);
+      domain.append(" AND self.id IN (" + idList + ")");
```

Comment:
`domain` is a `StringBuilder` so can you remove string concatenation with `+` (also at the start of the method)